app/controllers/base_assets_controller.rb

```diff
@@ -24,8 +24,8 @@ def base_asset_params
         asset[:redirect_url] = nil
       end
 
-      if asset.has_key?(:access_limited) && asset[:access_limited].empty?
-        asset[:access_limited] = []
+      if asset.has_key?(:access_limited_user_ids) && asset[:access_limited_user_ids].empty?
```

Comment:
I'm a bit confused why we're doing this, and why it doesn't apply to the new organisation array. Could you help me understand?

Reply:
> why we're doing this

This is was the existing functionality - keeping it to prevent breaking changes

> why it doesn't apply to the new organisation array

we are doing the same for organisations ids on lines 35-37